server/service/integration_mdm_test.go

```diff
@@ -5483,7 +5482,7 @@ func (s *integrationMDMTestSuite) TestOrbitConfigNudgeSettings() {
 
```

Comment:
would it be possible to add tests that simulate an iOS device getting the declaration? is our `mdmtest` package good to do so?

Reply:
Good idea. Will merge as is and will work on this when I get back.